CHANGELOG.txt

```diff
@@ -73,6 +73,7 @@ Changes:
    * add support for Chebyshev I/II, elliptic and Bessel filters alongside
      the default Butterworth filters (see #3294)
    * fix potential out of bounds array access in AR picker (see #1801)
+   * fix a small issue in evalresp linked to picky new gcc (see #3587)
```

Comment:
BTW, should this point to the issue or the PR (they happen to be the same digits...)?

Reply:
excellent question, on or the other is fine I guess. Looks like it was mostly the issues in the list, will change.